rust-mode-tests.el

```diff
@@ -2603,18 +2552,6 @@ Fontification needs to include this whole string or none of it.
     )
   )
 
-(ert-deftest rust-test-revert-hook-preserves-point ()
```

Comment:
This was a test for `rust--after-revert-hook` (I think specifically just for the call to `save-excursion` there), which is gone now.